ps06/CS2_2024_unicycle_discrete_stability.ipynb

```diff
@@ -0,0 +1,1323 @@
+{
```

Comment:
<p>Why is there a print in the solutions?</p><p>If you want them to check if the system is stable maybe do a separate task.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/idsc-frazzoli/CS2-2024-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>